src/Umbraco.Core/Persistence/Repositories/IContentTypeRepository.cs

```diff
@@ -29,4 +29,6 @@ public interface IContentTypeRepository : IContentTypeRepositoryBase<IContentTyp
     IEnumerable<string> GetAllContentTypeAliases(params Guid[] objectTypes);
 
     IEnumerable<int> GetAllContentTypeIds(string[] aliases);
+
+    PagedModel<Guid> GetChildren(Guid parentKey) => throw new NotImplementedException();
```

Comment:
I don't think this method is necessary ref. the comments in `ContentTypeReferenceService`